beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -72,11 +80,7 @@ lazy_static! {
     + types::ExecutionPayload::<MainnetEthSpec>::max_execution_payload_merge_size() // adding max size of execution payload (~16gb)
     + ssz::BYTES_PER_LENGTH_OFFSET; // Adding the additional ssz offset for the `ExecutionPayload` field
 
-    pub static ref SIGNED_BEACON_BLOCK_CAPELLA_MAX: usize = *SIGNED_BEACON_BLOCK_ALTAIR_MAX
-    + types::ExecutionPayload::<MainnetEthSpec>::max_execution_payload_capella_size() // adding max size of execution payload (~16gb)
```

Comment:
I don't think we should remove this allowance for the execution payload's transactions, should we? Otherwise we're adjusting the size down by a drastic amount, ~16GB.

Reply:
Actually it's not even 16GB, it's enormous: 1GB * 1M = 1000 TB.

Reply:
> I don't think we should remove this allowance for the execution payload's transactions, should we? Otherwise we're adjusting the size down by a drastic amount, ~16GB.

Great catch! I fat-fingered this 😬 

I've fixed this up now :)

Reply:
> I don't think we should remove this allowance for the execution payload's transactions, should we? Otherwise we're adjusting the size down by a drastic amount, ~16GB.

Great catch! I fat-fingered this 😬

I've fixed this up now :)